packages/botonic-react/src/components/whatsapp-media-carousel.tsx

```diff
@@ -0,0 +1,103 @@
+import { INPUT } from '@botonic/core'
+import React from 'react'
+
+import { renderComponent } from '../util/react'
+import { Message } from './message'
+
+type Parameters = TextParameter | CurrencyParameter | DateTimeParameter
+
+interface TextParameter {
+  type: 'text'
+  text: string
+}
+
+interface CurrencyParameter {
+  type: 'currency'
+  currency: {
+    fallback_value: string
+    code: string
+    amount_1000: number
+  }
+}
+
+interface DateTimeParameter {
+  type: 'date_time'
+  date_time: { fallback_value: string }
+}
+
+type CardButton = QuickReplyButton | UrlButton
+
+interface Button {
+  type: 'quick_reply' | 'url'
+  button_index?: number
+}
+
+interface QuickReplyButton extends Button {
+  payload: string
+}
+
+interface UrlButton extends Button {
+  url_variable: string
+}
+
+interface Card {
+  file_type: 'image' | 'video'
+  file_id: string
+  card_index?: number
+  body_parameters?: Parameters[]
+  buttons?: CardButton[]
+  extra_components?: Record<string, any>[]
```

Comment:
is this one not defined with camelCase?

Reply:
As mentioned in [this comment](https://github.com/hubtype/botonic/pull/2959#issuecomment-2597024889) the changes are applied in [this PR](https://github.com/hubtype/botonic/pull/2965).